includes/wpmenucart-settings.php

```diff
@@ -655,7 +655,9 @@ public function select_element_callback( $args ) {
 		$disabled = (isset( $args['disabled'] )) ? ' disabled' : '';
 		
 		$html  = sprintf( '<select name="%1$s[%2$s]" id="%1$s[%2$s]"%3$s>', esc_attr( $menu ), esc_attr( $id ), esc_attr( $disabled ) );
-		$html .= sprintf( '<option value="%s"%s>%s</option>', '0', selected( $current, '0', false ), '' );
+		if ( 'shop_plugin' === $args['id'] ) {
+			$html .= sprintf( '<option value="">%s</option>', __( 'None', 'wp-menu-cart' ) );
```

Comment:
Maybe **Select** instead of **None** would fit better?

Reply:
@alexmigf what do you think about `Select a choice…` or `Select an option…`? If you like one of them, please tell me what you like the most.

Reply:
@YordanSoares  as you prefer, both options are valid for me.

Reply:
Great! Updated here [d6d13bd](https://github.com/wpovernight/wp-menu-cart/pull/50/commits/d6d13bddf8d2763edd6aa0877049295d6299a0a3)